templates/common/baremetal/files/NetworkManager-static-dhcpv6.yaml

```diff
@@ -19,6 +19,14 @@ contents:
         exit 0
     fi
 
+    # We don't want this to run before OVNKubernetes creates its bridge. If we
+    # see the default CONNECTION_ID we know to wait.
+    if [ "$CONNECTION_ID" == "Wired Connection" ]
```

Comment:
Should we check if the networking sdn is ovn kubernetes like we do in IPv4 (in case somebody uses a third party SDN, since we know that openshift-sdn is not an option)?

Reply:
I'm thinking not. We know that things break if we modify the Wired Connection because unlike the v4 version we don't create a new connection here. I think it's better for us to refuse to run this with untested SDNs than to hope it works.

Reply:
alright